### ISSUE TITLE
No unicode support for pages

### ISSUE THREAD

**Issue by wh1t3h47** [vikejs/vike#1658]
### Description

Hello,

Steps to Reproduce:
`vite.config.js`
```jsx
import { defineConfig } from "vite";
import react from "@vitejs/plugin-react";
import tailwindcss from "tailwindcss";
import vike from "vike/plugin";

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react(), vike({ prerender: true })],
  css: {
    postcss: {
      plugins: [tailwindcss()],
    },
  },
});
```

`src/pages/+Layout.jsx`

```jsx
import React from "react";
import PropTypes from 'prop-types'
import "../index.css";
import { NavPageTemplate } from "../templates/NavPageTemplate/index.jsx";

/**
 * @type {React.FunctionComponent<React.PropsWithChildren<{}>>}
 */

export const Layout = ({ children }) => {
  return (
    <React.StrictMode>
      <NavPageTemplate />
      { children }
    </React.StrictMode>
  );
};

Layout.propTypes = {
  children: PropTypes.node.isRequired,
}
```

`src/pages/config.js`
```js
import vikeReact from 'vike-react/config'

// https://vike.dev/config
export default {
  // https://vike.dev/head
  favicon: '/vite.svg',
  extends: [vikeReact],
}
```

Now create a page with some unicode path
```bash
mkdir src/pages/canções/
touch src/pages/canções/+Page.jsx
```

```bash
10:27:22 PM [vike@0.4.172][config][Bug] You stumbled upon a Vike bug. Go to https://github.com/vikejs/vike/issues/new and copy-paste this error. A maintainer will fix the bug (usually under 24 hours). Debug info (for Vike maintainers; you can ignore this): Not a posix path: "src/pages/\303\263example/+Page.jsx"
    at assertPosixPath (file:///home/skid/My_Code/test/node_modules/vike/dist/esm/utils/filesystemPathHandling.js:15:5)
    at isTemporaryBuildFile (file:///home/skid/My_Code/test/node_modules/vike/dist/esm/node/plugin/plugins/importUserCode/v1-design/getVikeConfig/transpileAndExecuteFile.js:304:5)
    at file:///home/skid/My_Code/test/node_modules/vike/dist/esm/node/plugin/plugins/importUserCode/v1-design/getVikeConfig/crawlPlusFiles.js:39:37
    at Array.filter (<anonymous>)
    at crawlPlusFiles (file:///home/skid/My_Code/test/node_modules/vike/dist/esm/node/plugin/plugins/importUserCode/v1-design/getVikeConfig/crawlPlusFiles.js:39:19)
    at findPlusFiles (file:///home/skid/My_Code/test/node_modules/vike/dist/esm/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:769:19)
    at loadInterfaceFiles (file:///home/skid/My_Code/test/node_modules/vike/dist/esm/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:80:23)
    at loadVikeConfig (file:///home/skid/My_Code/test/node_modules/vike/dist/esm/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.js:254:40)
    ```

**Comment by wh1t3h47** [comment]
"vike-react": "^0.4.10"
    "vike": "^0.4.172",
    "vite": "^5.2.0"
    "react": "^18.2.0",

**Comment by wh1t3h47** [comment]
It works fine with no unicode-like path or folders

**Comment by brillout** [comment]
Indeed I could reproduce, I'll see if/how such paths can be supported. Use a Route String instead in the meantime.

**Comment by brillout** [comment]
Fix pre-released in `0.4.172-commit-ea2b76c`.

**Comment by wh1t3h47** [comment]
Awesome! Thanks for the fix, that was quick!
